libc/test/src/math/smoke/CMakeLists.txt

```diff
@@ -3618,6 +3618,45 @@ add_fp_unittest(
     libc.src.math.atan2
 )
 
+add_fp_unittest(
+  scalbln_test
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    scalbln_test.cpp
+  HDRS
+    ScalbnTest.h
+  DEPENDS
+    libc.src.math.scalbln
+    libc.src.__support.FPUtil.fp_bits
+)
```

Comment:
The `scalbn*` tests' dependencies should be updated to match the includes of LdExpTest.h, which is itself included by ScalbnTest.h. The `ldexp*` tests' dependencies are correct.